docs/references/protocol/transactions/types/oracleset.md

```diff
@@ -0,0 +1,97 @@
+---
+html: OracleSet.html 
+parent: transaction-types.html
+blurb: Create or update an existing price oracle.
+labels:
+  - Oracle
+status: not_enabled
+---
+# OracleSet
+_(Requires the [PriceOracle amendment][] {% not-enabled /%})_
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/app/tx/impl/SetOracle.cpp "Source")
+
+Creates a new `Oracle` ledger entry or updates the fields of an existing one, using the Oracle ID.
+
+The oracle provider must complete these steps before submitting this transaction:
+
+1. Create or own the XRPL account in the `Owner` field and have enough XRP to meet the reserve and transaction fee requirements.
+2. Publish the XRPL account public key, so it can be used for verification by dApps.
+3. Publish a registry of available price oracles with their unique `OracleDocumentID`.
```

Comment:
I don't think this list of requirements makes sense here. The first point is literally a requirement for sending any transaction that creates a ledger entry, and the other two points are off-chain steps that can't possibly be hard requirements for sending the transaction—meaning that the user could do them later if at all, and the ledger itself would neither know nor care.